### ISSUE TITLE
Citra crashes when settings file is missing (or old)

### ISSUE THREAD

**Issue by LittleWhite-tb** [citra-emu/citra#1944]
Hello,

I encounter a crash with citra on Linux.
```
[   0.001852] Debug <Critical> common/emu_window.cpp:operator():96: Assertion Failed!
```

![crash](https://cloud.githubusercontent.com/assets/8618754/16709049/ef6874d2-4605-11e6-825a-165f83640c82.png)

We can easely reproduce it by just removing the settings file in .config/citra-emu/qt-config.ini
It also happen if the file is old (coming from a old citra version).

**Comment by yuriks** [comment]
Can you upload your qt-config.ini file which reproduces this crash? (You can edit any paths in it if you want to "anonymize" it to remove file history, etc.)

**Comment by yuriks** [comment]
Very likely can be considered a dupe of #1205 (even though the crash is in a separate location).

**Comment by LittleWhite-tb** [comment]
I could quick fix the issue by adding a check in GRenderWindow::OnFramebufferSizeChanged() in order to avoid a size of 0.
But, at the moment, I consider it just a quick fix (not a good fix).

About the config file. One way to reproduce it is to remove the config file (renaming/deleting it).
Otherwise, my old config file was :

[old.ini.zip](https://github.com/citra-emu/citra/files/355931/old.ini.zip)

**Comment by LittleWhite-tb** [comment]
I think, this one should be closed since #1974 is merged.
